controller/app/models/cloud_user.rb

```diff
@@ -249,6 +249,7 @@ def default_capabilities
       "max_untracked_addtl_storage_per_gear" =>  Rails.application.config.openshift[:default_max_untracked_addtl_storage_per_gear],
       "max_tracked_addtl_storage_per_gear" =>  Rails.application.config.openshift[:default_max_tracked_addtl_storage_per_gear],
       "private_ssl_certificates" => Rails.application.config.openshift[:default_private_ssl_certificates],
+      "node_df_buffer" => Rails.application.config.openshift[:default_node_df_buffer],
```

Comment:
I don't think we need the changes in this file.  The setting is cluster-wide and has nothing to do with individual users or their capabilities.